x-pack/solutions/security/plugins/security_solution/server/endpoint/services/actions/clients/crowdstrike/crowdstrike_actions_client.test.ts

```diff
@@ -50,9 +51,20 @@ describe('CrowdstrikeActionsClient class', () => {
     classConstructorOptions = CrowdstrikeMock.createConstructorOptions();
     connectorActionsMock = classConstructorOptions.connectorActions;
     crowdstrikeActionsClient = new CrowdstrikeActionsClient(classConstructorOptions);
-    classConstructorOptions.esClient.search.mockResolvedValueOnce(
-      CrowdstrikeMock.createEventSearchResponse()
-    );
+
+    const esClientSearchCallback = classConstructorOptions.esClient.search.getMockImplementation();
+    classConstructorOptions.esClient.search.mockImplementation(async (...args) => {
+      const options = args[0];
+      if (options?.index?.[0] === 'logs-crowdstrike*') {
```

Comment:
How does this comparison to `logs-crowdstrike*` work? I mean the asterisk.

Reply:
If the request to the `search()` has an `index` of `['logs-crowdstrike*']` (exact match), then the mock response below is sent.

Reply:
Oh, so * is included in the exact match. I wasn't expecting this :P Thanks for clarification!